test/rauc.t

```diff
@@ -202,6 +202,11 @@ test_expect_success "rauc info" "
     info $SHARNESS_TEST_DIRECTORY/good-bundle.raucb
 "
 
+test_expect_success "rauc info dump-cert unverified" "
+  rauc --no-verify --dump-cert \
+    info $SHARNESS_TEST_DIRECTORY/good-bundle.raucb
```

Comment:
Shouldn't we test this on bad bundle?

Reply:
The relevant check for this patch is that --dump-cert works without verification. That verification is note performed here is assured as we do not provide a keyring.

However, despite a bit unrelated, I now added a check to assure that rauc info fails verification upon invalid signature.